extensions/buildpack-registry.md

```diff
@@ -92,11 +84,21 @@ The fields are defined as follows:
 * `addr` - the address of the image stored in a Docker Registry (ex. `"docker.io/jkutner/lua@sha256:abc123"`). The image reference must use `@digest` and not use `:tag`.
 
 
-An example of what this may look like for a single buildpack file:
+## Example
+
+A buildpack with the ID `example/java` is stored in the registry under this file structure:
+
+```
+ja
+└── va
+    └── example_java
+```
+
+The `example_java` file contains the following data:
 
 ```
-{"ns":"example","name":"ruby","version":"0.1.0","yanked":false,"addr":"docker.io/hone/ruby-buildpack@sha256:a9d9038c0cdbb9f3b024aaf4b8ae4f894ea8288ad0c3bf057d1157c74601b906"}
-{"ns":"example","name":"ruby","version":"0.2.0","yanked":false,"addr":"docker.io/hone/ruby-buildpack@sha256:2560f05307e8de9d830f144d09556e19dd1eb7d928aee900ed02208ae9727e7a"}
-{"ns":"example","name":"ruby","version":"0.2.1","yanked":false,"addr":"docker.io/hone/ruby-buildpack@sha256:74eb48882e835d8767f62940d453eb96ed2737de3a16573881dcea7dea769df7"}
-{"ns":"example","name":"ruby","version":"0.3.0","yanked":false,"addr":"docker.io/hone/ruby-buildpack@sha256:8c27fe111c11b722081701dfed3bd55e039b9ce92865473cf4cdfa918071c566"}
+{"ns":"example","name":"java","version":"0.1.0","yanked":false,"addr":"docker.io/cnb-example/java-buildpack@sha256:a9d9038c0cdbb9f3b024aaf4b8ae4f894ea8288ad0c3bf057d1157c74601b906"}
+{"ns":"example","name":"java","version":"0.2.0","yanked":false,"addr":"docker.io/cnb-example/java-buildpack@sha256:2560f05307e8de9d830f144d09556e19dd1eb7d928aee900ed02208ae9727e7a"}
+{"ns":"example","name":"java","version":"0.2.1","yanked":false,"addr":"docker.io/cnb-example/java-buildpack@sha256:74eb48882e835d8767f62940d453eb96ed2737de3a16573881dcea7dea769df7"}
+{"ns":"example","name":"java","version":"0.3.0","yanked":false,"addr":"docker.io/cnb-example/java-buildpack@sha256:8c27fe111c11b722081701dfed3bd55e039b9ce92865473cf4cdfa918071c566"}
```

Comment:
If our example is going to point at a real registry instead of something like `example.com` do we want to reference an org we own (`buildpacksio` or `cnbs`) on the extreme off-chance somebody registers `cnb-example`?

Reply:
Agreed.  Don't even bother to put anything there, just take ownership of the repository.

Reply:
@ekcasey @nebhale i changed it to `docker.io/cnbs/fake-buildpack` and the shas are fake. does that look good?

Reply:
lgtm